api/sbom.proto

```diff
@@ -346,3 +349,9 @@ enum Purpose {
     SPECIFICATION = 27; // Specification purpose. (SPDX3.0)
     TEST = 28; // Test purpose. (SPDX3.0)
 }
+
+// Property type
+message Property {
+    string name = 1;    // Name of the property
+    string data = 2;    // Property value
+}
```

Comment:
Do you still get the name collision if you nest this message inside the `Node` message and set `data` back to `value`?

Reply:
Yeah, nesting it only makes the `Property` struct in go to be named  `Node_Property` but its `Value` property will still clash with the `Value()` func from the value scanner.